content/guides/output-management.md

```diff
@@ -221,6 +222,20 @@ __webpack.config.js__
   };
 ```
 
+__src/index.html__
```

Comment:
Just go ahead and take this piece out then and commit it, then I can approve this and merge it in ;)

Thanks for helping!